Controllers/Controller.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * PageController class file
+ *
+ * PHP Version 7.2
+ *
+ * @category Controllers
+ * @package  Controllers
+ * @author   Hervé Boulangué <h.boulangue@gmail.com>
+ * @license  http://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     http://blog.local/
+ */
+
+/**
+ * Controller for the rendering of Twig's template
```

Comment:
Base Controller with useful methods that every controller of the application should extend

Reply:
Commentaire à mettre juste au dessus de `class Controller`